http_example_test.go

```diff
@@ -101,8 +101,12 @@ func Example_getTokenViaHTTP() {
 
 	// Read the token out of the response body
 	buf := new(bytes.Buffer)
-	io.Copy(buf, res.Body)
-	res.Body.Close()
+	if _, err = io.Copy(buf, res.Body); err != nil {
+		fatal(err)
+	}
+	if err = res.Body.Close(); err != nil {
```

Comment:
Not very useful to panic in this situation.
An error when closing a response.Body can be ignored - the client has gone anyway.

This is an example where using golangci-lint does not improve the code, 
but makes it unnecessarily convoluted.

The Go stdlib wil notpass golangci-lint.